platform/darwin/test/MGLDocumentationExampleTests.swift

```diff
@@ -29,6 +29,14 @@ class MGLDocumentationExampleTests: XCTestCase, MGLMapViewDelegate {
     var styleLoadingExpectation: XCTestExpectation!
     static let styleURL = Bundle(for: MGLDocumentationExampleTests.self).url(forResource: "one-liner", withExtension: "json")!
 
+    // Mock MGLOfflineStorage singleton so that it doesn't start long-running tasks that could interfere with other tests.
+    fileprivate class MGLOfflineStorageMock {
+        static let shared = MGLOfflineStorageMock()
+        func addPack(for: MGLOfflineRegion, withContext: Data, completionHandler: MGLOfflinePackAdditionCompletionHandler? = nil) {
+            XCTAssert(MGLOfflineStorage.shared.responds(to: #selector(MGLOfflineStorage.shared.addPack(for:withContext:completionHandler:))))
+        }
```

Comment:
Even though this is a documentation test - I think we should call the completion block (if provided) with an "error". Any reason why we shouldn't?

Reply:
I don’t see a good way to test within `testMGLOfflinePack()`’s completion handler (since it’s entirely encapsulated by the example block, which can’t be split into multiple parts), so adding this is mostly informative... but I will add it.